CHANGES.md

```diff
@@ -0,0 +1,86 @@
+# Revision history for sDDF
+
+## Release 0.5.0
+
+### General
+
+* Move to Microkit 1.4.0.
+    * Previously a development version of Microkit was used, now we use
+      an official release for everything.
+* Add QEMU support to the timer and echo server examples.
+* Remove dependency on a full libc, all the provided components use our own
+  utility functions now.
+* Moved all components and examples to use a 'Makefile snippets' system
+  to make it easier to put together sDDF components in a system.
+* Introduced snippets for the Zig build system, as an alternative to Make.
+    * You can find instructions for using the Zig build system in each example's
+      README.
```

Comment:
> You can find instructions for using the Zig build system in each example's README.

Instructions for using the Zig build system can be found in each example's README.